pandas/tests/reductions/test_reductions.py

```diff
@@ -82,19 +113,27 @@ def test_nanops(self):
         obj = Index([np.nan, 1, np.nan, 2])
         assert obj.argmin() == 1
         assert obj.argmax() == 3
+        assert obj.argmin(skipna=False) == -1
+        assert obj.argmax(skipna=False) == -1
 
         obj = Index([np.nan])
         assert obj.argmin() == -1
         assert obj.argmax() == -1
+        assert obj.argmin(skipna=False) == -1
+        assert obj.argmax(skipna=False) == -1
 
         obj = Index([pd.NaT, datetime(2011, 11, 1), datetime(2011, 11, 2),
                      pd.NaT])
         assert obj.argmin() == 1
         assert obj.argmax() == 2
+        assert obj.argmin(skipna=False) == -1
+        assert obj.argmax(skipna=False) == -1
```

Comment:
really? this doesn't seem right

Reply:
What would you expect it to be?  For Series idxmin/idxmax would give np.nan